test/integration/patients/sidebar/action-sidebar.js

```diff
@@ -1886,6 +1876,72 @@ context('action sidebar', function() {
       .and('contain', 'You are not able to change settings on this action.');
   });
 
+  specify('action with work:authored:delete permission', function() {
+    cy
+      .routesForPatientAction()
+      .routeCurrentClinician(fx => {
+        fx.data.relationships.role = { data: { id: '77777' } };
+        fx.data.relationships.team = { data: { id: '11111', type: 'teams' } };
+
+        return fx;
+      })
+      .routePatient(fx => {
+        fx.data.id = '1';
+
+        return fx;
+      })
+      .routeAction(fx => {
+        fx.data = {
+          id: '1',
+          attributes: {
+            name: 'Owned by the team',
+          },
+          relationships: {
+            author: { data: { id: '11111', type: 'clinicians' } },
+            owner: { data: { id: '11111', type: 'teams' } },
+            state: { data: { id: '33333' } },
+          },
+        };
+
+        return fx;
+      })
+      .routePatientActions(fx => {
+        fx.data = _.sample(fx.data, 2);
+
+        fx.data[0].id = '1';
+        fx.data[0].attributes.name = 'Owned by the team';
+        fx.data[0].attributes.updated_at = testTsSubtract(1);
+        fx.data[0].relationships.author = { data: { id: '11111', type: 'clinicians' } };
+        fx.data[0].relationships.state = { data: { id: '33333' } };
+        fx.data[0].relationships.owner = { data: { id: '11111', type: 'teams' } };
+
+        fx.data[1].id = '2';
+        fx.data[1].attributes.name = 'Owned by non team member';
+        fx.data[1].attributes.updated_at = testTsSubtract(2);
+        fx.data[1].relationships.state = { data: { id: '33333' } };
+        fx.data[1].relationships.owner = { data: { id: '22222', type: 'clinicians' } };
```

Comment:
Do we need this action? I don't think it's being used in the test from what I can tell...

Or I guess we could change it to a `Not authored by current user` action:

```js
fx.data[1].id = '2';
fx.data[1].attributes.name = 'Not authored by current user';
fx.data[1].relationships.author = { data: { id: '22222', type: 'clinicians' } };
fx.data[1].attributes.updated_at = testTsSubtract(2);
fx.data[1].relationships.state = { data: { id: '33333' } };
fx.data[1].relationships.owner = { data: { id: '22222', type: 'clinicians' } };
```

And test that the menu doesn't show for that action since the author isn't the current user.

But probably not necessary.